### PR TITLE
Codemod for removing `<ClientSideSuspense>` thunk

### PR DESCRIPTION
**Pull request by nvie**
Changes:

```tsx
<ClientSideSuspense>
  {() => <MyApp />}
</ClientSideSuspense>
```

To:

```tsx
<ClientSideSuspense>
  <MyApp />
</ClientSideSuspense>
```

### CODE REVIEW

File: packages/liveblocks-codemod/README.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ are using the global types).
 npx @liveblocks/codemod@latest remove-unneeded-type-params
 ```
 
+#### `remove-clientsidesuspense-thunk`
```
Comment (marcbouchenoire) [inline packages/liveblocks-codemod/README.md:45]: Maybe `simplify-client-side-suspense-children`?

We use dashes in all other codemods, and personally, I didn't know the word "thunk" before you used it, is it well known?

**Review by marcbouchenoire — CHANGES_REQUESTED** [review]
I just left a comment about the name, but the codemod itself looks good!

File: packages/liveblocks-codemod/README.md (hunk 1 above)
Comment (nvie) [reply packages/liveblocks-codemod/README.md:45]: I think Dan Abramov used it extensively in early React posts and videos, that's where I got it from. But I don't mind the name you suggested, will update! 👍

Comment (nvie) [reply packages/liveblocks-codemod/README.md]: Done in 3d084f72f72070fd728227a59cc3c6669700d557.

File: packages/liveblocks-codemod/bin/cli.ts [hunk 2]
```diff
@@ -45,8 +45,8 @@ const TRANSFORMER_INQUIRER_CHOICES = [
     value: "live-list-constructor",
   },
   {
-    name: "remove-clientsidesuspense-thunk: Removes any thunks from <ClientSideSuspense>'s children.",
-    value: "remove-clientsidesuspense-thunk",
+    name: "simplify-client-side-suspense-children: Removes the function-style body from <ClientSideSuspense>'s children.",
```
Comment (marcbouchenoire) [inline packages/liveblocks-codemod/bin/cli.ts:48]: `Removes the function-style body` → `Removes any function-style body` like in the `README.md`

**Review by marcbouchenoire — APPROVED** [review]

**Pull request merged**